db/schema.rb

```diff
@@ -215,19 +215,26 @@
     t.string   "cardholder_name"
     t.string   "card_debit"
     t.string   "card_last_4"
```

Comment:
We kind of agreed to move that to a new PR once we'll be working on the member management sprint again. We can do that now for the GoCardless tables and I can add that later for Braintree, as well.